scripts/setup-kind-multicluster.sh

```diff
@@ -7,24 +7,81 @@
 #    - kustomize
 #    - kind
 
-# TODO add check to make sure gnu getopt is installed. This can be done with the -T option
-# of getopt.
-
 set -e
 
-OPTS=$(getopt -o h --long clusters:,cluster-names:,kind-node-version:,kind-worker-nodes:,help -n 'create-kind-clusters' -- "$@")
+registry_name='kind-registry'
+registry_port='5000'
 
-eval set -- "$OPTS"
+num_clusters=1
+cluster_names="kind"
+kind_node_version="v1.21.2"
+kind_worker_nodes=3
 
-function help() {
-  echo
-  echo "Syntax: create-kind-clusters.sh [options]"
-  echo "Options:"
-  echo "clusters           The number of clusters to create."
-  echo "cluster-names      A comma-delimited list of cluster names to create. Takes precedence over clusters option."
-  echo "kind-node-version  The image version of the kind nodes."
-  echo "kind-worker-nodes  The number of worker nodes to deploy."
-}
+while test $# -gt 0; do
+  case "$1" in
+    --clusters)
+      shift
+      if test $# -gt 0; then
+        export num_clusters=$1
+      else
+        echo "no num clusters specified"
+        exit 1
+      fi
+      shift
+      ;;
+    --cluster-names)
+      shift
+      if test $# -gt 0; then
+        export cluster_names=$1
+      else
+        echo "no clusters names specified"
+        exit 1
+      fi
+      shift
+      ;;
+    --kind-node-version)
+      shift
+      if test $# -gt 0; then
+        export kind_node_version=$1
+      else
+        echo "no kind node version specified"
+        exit 1
+      fi
+      shift
+      ;;
+    --kind-worker-nodes)
+      shift
+      if test $# -gt 0; then
+        export kind_worker_nodes=$1
+      else
+        echo "no kind worker nodes specified"
+        exit 1
+      fi
+      shift
+      ;;
+    -h|--help) 
+      echo
+      echo "Syntax: create-kind-clusters.sh [options]"
+      echo "Options:"
+      echo "clusters           The number of clusters to create."
+      echo "cluster-names      A comma-delimited list of cluster names to create. Takes precedence over clusters option."
+      echo "kind-node-version  The image version of the kind nodes."
+      echo "kind-worker-nodes  The number of worker nodes to deploy."
+      exit
+      ;;
+    --)
+      shift;
+      break
+      ;;
+    *) 
+      break
+      ;;
+  esac
+done
+
+# OPTS=$(getopt -o h --long clusters:,cluster-names:,kind-node-version:,kind-worker-nodes:,help -n 'create-kind-clusters' -- "$@")
+
+# eval set -- "$OPTS"
```

Comment:
small nit: Just remove these commented lines.

Reply:
right, thought I had done it 😅  thanks!